src/AppBundle/Resources/views/Report/Formatted/Accounts/_final_summary.html.twig

```diff
@@ -16,7 +16,7 @@
             </tr>
             <tr>
                 <td>Money out:</td>
-                <td class="grey numeric">&pound;{{ report.moneyOutTotal | money_format}}</td>
+                <td class="grey numeric">&pound;{{ (report.moneyOutTotal + report.profDeputyTotalCosts) | money_format}}</td>
```

Comment:
this will need a check on hasSection otherwise when we feature flag to disable the section, this will break